Sources/ClientRuntime/Retries/ExponentialBackoffStrategy.swift

```diff
@@ -0,0 +1,26 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import func Foundation.pow
+import struct Foundation.TimeInterval
+
+struct ExponentialBackoffStrategy: RetryBackoffStrategy {
```

Comment:
Calculates time delays for retry, using an exponential formula capped with a flat maximum delay.